packages/reapit-connect-management/src/utils/headers.ts

```diff
@@ -0,0 +1,10 @@
+import { BASE_HEADERS } from '../constants/api'
+import { reapitConnectBrowserSession } from '../core/connect-session'
+
+export const genPlatformHeaders = async () => {
+  const connectSession = await reapitConnectBrowserSession.connectSession()
+  return {
+    ...BASE_HEADERS,
+    Authorization: (connectSession && (connectSession?.accessToken as string)) ?? '',
```

Comment:
Headers should try catch if the accessToken is not present. An empty string returned will throw an error further downstream